source/code-generation/query-converter/convert-views.txt

```diff
@@ -5,7 +5,193 @@ Convert Views
 =============
 
 .. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+  :local:
+  :backlinks: none
+  :depth: 1
+  :class: singlecol
+
+You can convert SQL views to MongoDB code with the query
+converter. The query converter considers the mapping rules and schema
+transformations defined in your project when converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+.. include:: /includes/fact-query-converter-disclaimer.rst
+
+Steps
+-----
+.. procedure::
+  :style: normal
```

Comment:
Vale raised a false positive for this line. We created https://jira.mongodb.org/browse/DOCSP-34343 to investigate.